web-common/src/features/explores/selectors.ts

```diff
@@ -131,123 +124,32 @@ export async function fetchExploreSpec(
   }
 
   const defaultExplorePreset = getDefaultExplorePreset(
-    exploreResource.explore.state?.validSpec ?? {},
+    exploreSpec,
     fullTimeRange,
   );
+  const { partialExploreState: initExploreState, errors } =
```

Comment:
Let's be more clear about where this state comes from: `defaultExploreYAMLState`, `exploreYAMLDefaultsState`, `stateFromExploreYAMLDefaults`? We should at least use "default"/"defaults" b/c it's the term used in the Explore YAML.